addons/l10n_au/data/l10n_au_chart_data.xml

```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0" encoding="utf-8"?>
 <odoo>
     <record id="l10n_au_chart_template" model="account.chart.template">
-        <field name="name">Australian Tax and Account Chart Template (by Willow IT)</field>
+        <field name="name">Australian Tax and Account Chart Template (by Willdoo IT)</field>
```

Comment:
Unfortunately it's no longer our policy to put this kind of attribution in the chart template name. In saas-16.2 onward it will no longer be there regardless.

I really hope it's not rude. I'm 100% aware that this publicity is part of your business. What we can do is make this change in 15.0 onward, but it can't go into saas-16.2 onward.